backend/src/config/metadata.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  APPLICATION_NAME: 'ocelot.social',
+  APPLICATION_SHORT_NAME: 'ocelot',
+  APPLICATION_DESCRIPTION: 'Ocelot Social Network',
+  ORGANIZATION_NAME: 'busFaktor() e.V.',
+  ORGANIZATION_JURISDICTION: 'Augsburg',
```

Comment:
```suggestion
  ORGANIZATION_JURISDICTION: 'City of Angles',
```

Reply:
By the seven hells. City of Angels then.